docs/ABOUT.md

```diff
@@ -1,5 +1,23 @@
-C# is a modern, object oriented language developed and maintained by Microsoft. The learning curve is small, reduced even if the developer has experience with other 'C-Style' languages.
+C# is a multi-paradigm statically-typed programming language with object-oriented, declarative, functional, generic, lazy integrated querying features and type inference. 
```

Comment:
Please add a comma between "multi-paradigm" and "statically-typed"

Reply:
Please add a comma between "lazy" and "integrated querying"

Reply:
First comma yes, the second was deliberate as it is only LINQ that is lazy (although one could add Tasks etc. I suppose).